.github/actions/test-type-vars/action.yml

```diff
@@ -71,6 +71,13 @@ runs:
         echo "TEST_CHART_DIR=${charts_base_dir}/${{ inputs.chart-dir }}" | tee -a $GITHUB_ENV
         echo "ABSOLUTE_TEST_CHART_DIR=${GITHUB_WORKSPACE}/${CHART_PATH}" | tee -a $GITHUB_ENV
 
+        # configuring TEST_HELM_EXTRA_ARGS
         if [[ "${{ inputs.enterprise }}" == "true" ]]; then
-          echo "TEST_HELM_EXTRA_ARGS=--values ${charts_base_dir}/${{ inputs.chart-dir }}/values-enterprise.yaml" | tee -a $GITHUB_ENV
+          helm_extra_args="--values ${charts_base_dir}/${CHART_DIR}/values-enterprise.yaml"
```

Comment:
Using the env var `CHART_DIR` make the code more complex with no added value as it's not defend in this action file.

Maybe it's better to use `TEST_CHART_DIR`, which is echoed and just needs to be defined.

Reply:
The relative paths are super confusing I added some comments to ease confusion. Hope this is okay